_source/logzio_collections/_metrics-sources/jmx.md

```diff
@@ -157,15 +151,13 @@ consider using [jmx2logzio with Jolokia](#with-a-jolokia-agent-config) instead.
 
 ## jmx2logzio + Jolokia agent setup
 
-If you experience issues such as port conflicts for Docker containers,
-you can run a Jolokia agent to act as a go-between for sending metrics from your app to Logz.io.
-In this configuration, jmx2logzio forwards metrics to Jolokia,
-which sends those metrics to Logz.io.
+If you to poll metrics from a remote source, you can use Jolokia agent + Jmx2logzio
+In this configuration, Jolokia exposes the metrics as REST, and jmx2logzio reads it from them and sends it to Logz.io
```

Comment:
Not sure what you mean here—can you clarify?

Reply:
The previous description was wrong and misleading. What's actually happens is Jolokia polls the metrics from the target app, expose them as REST API, and once every x seconds jmx2logzio call this api and receive the metrics as answer

Reply:
```suggestion
In this configuration, Jolokia exposes the metrics through an API, which jmx2logzio reads and sends to Logz.io.
```